README.md

```diff
@@ -1,5 +1,46 @@
+[![Build Status](https://travis-ci.org/ARGOeu/argo-messaging.svg?branch=devel)](https://travis-ci.org/ARGOeu/argo-messaging)
 # ARGO Messaging
 
+## Development
+
+1. Install Golang and bzr library
+2. Create a new work space:
+
+      `mkdir ~/go-workspace`
+      `export GOPATH=~/go-workspace`
+
+  You may add the last `export` line into the `~/.bashrc` or the `~/.bash_profile` file to have `GOPATH` environment variable properly setup upon every login.
+
+3. Get the latest version
+
+      `go get github.com/ARGOeu/argo-messaging`
+
+4. Get dependencies:
+
+   Argo-messaging uses godep tool for dependency handling.
+   To install godep tool issue:
+
+      `go get github.com/tools/godep`
+      `godep update ...`
+      `godep restore`
+
```

Comment:
I tried to follow this step by step instructions. I think that steps 3 and 4 should be changed in sequence. 

Also I see the update & restore commands are the other way around in the travis and spec files. Should this be the case also here?